restapi/serializers.py

```
@@ -63,13 +70,20 @@ def update(self, instance, validated_data):
                     for user_expense in user_expenses
                 ],
             )
-        instance.save()
+        try:
```

Is this change really helpful? Think about what you're doing here. Earlier, if there was an exception while saving the instance, it would break the normal flow of execution and send an exception to the client. In your case, the exception is caught, it's logged and then the execution will proceed as if nothing happened and will try to return an instance object that actually wasn't saved. Even though an error happened and was logged, the caller will have no idea that something failed. This is almost like swallowing an exception.